src/panels/config/info/system-health-card.ts

```diff
@@ -104,6 +127,27 @@ class SystemHealthCard extends LitElement {
     }
   }
 
+  private _copyInfo(): void {
+    // We have to copy title text and content separately, because
+    // copying the whole <ha-card> would also copy the tooltip text.
```

Comment:
No longer?

Reply:
No, with the move to using `<h>` in `<ha-card>` this was no longer needed. Comment removed.